examples/compass.py

```diff
@@ -1,7 +1,16 @@
+"""
+    compass.py
+    ~~~~~~~~~~
+
+    Creates a compass.
+
+    To calibrate the compass, lie it flat and spin it horizontally. For best
```

Comment:
@blmorris You are absolutely correct. It is "lay it flat". Updating. Great catch.